api/src/bin/main.rs

```diff
@@ -5,9 +5,8 @@ use axum::{
     routing::{delete, get, post, put},
```

Comment:
So, for some reason I thought you were going to make a more opinionated proxy for axum in platform and use that here in the API (aqum?).

Reply:
That could be done for sure. I haven't worked on the API for a while TBH since priorities shifted to ingest/enrichment.

Reply:
OK, then where was that bucket for "ideas" rather than work like Jira?  This idea should go there so we can distill it and talk about it later.   I'm going to figure it out.

Reply:
I created a Trello Board with Actionable ideas.  It's mine for now and I'll add to it if I need to